lib/saluki-components/src/transforms/host_tags/mod.rs

```diff
@@ -124,3 +129,44 @@ impl SynchronousTransform for HostTagsEnrichment {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+
+    use std::{
+        num::NonZeroUsize,
+        time::{Duration, Instant},
+    };
+
+    use bytesize::ByteSize;
+    use saluki_context::{Context, ContextResolverBuilder};
+    use saluki_event::metric::Metric;
+
+    use super::*;
+
+    #[tokio::test]
+    async fn test_host_tags_enrichment() {
+        let context_resolver = ContextResolverBuilder::from_name("host_tags")
+            .expect("resolver name is not empty")
+            .with_interner_capacity_bytes(NonZeroUsize::new(ByteSize::kib(64).as_u64() as usize).unwrap())
+            .build();
```

Comment:
This can just be `let context_resolver = ContextResolverBuilder::for_tests().build();` since we don't care about the resolver name of the interner capacity, as the default capacity is more than enough for this test.